rio_tiler_pds/cbers/aws/cbers4.py

```diff
@@ -2,7 +2,7 @@
 
 from typing import Dict, Type
 
-import attr
+import attr  # type: ignore
```

Comment:
Why is this needed?

Reply:
is this related to the mypy bump?

Reply:
Yes this was related to the `mypy` bump which was needed for [M1 support](https://stackoverflow.com/questions/71291254/pre-commit-giving-error-with-mirrors-mypy-how-do-i-fix-it
). Mypy versions <0.900 depend on typed-ast < 1.5.0 which does not support M1.

Running the new `mypy` version via pre-commit caused a incompatibility error with `attrs`. I've removed the `ignore` and added `attrs` as an additional dependency which fixes the error.